<launch>
	<arg name="show_image"          default="true" />
	<node name="nuitrack_node" type="nuitrack_node.py" pkg="mild_hri_ros" respawn="True"/>
	
    <node pkg="nodelet" type="nodelet" args="manager" name="standalone_nodelet" output="screen"/>
	<node pkg="nodelet" type="nodelet" name="depth_image_proc" args="load depth_image_proc/point_cloud_xyzrgb standalone_nodelet --no-bond">
			<remap from="rgb/camera_info" to="camera_info"/>
			<remap from="rgb/image_rect_color" to="image_color"/>
			<remap from="depth_registered/image_rect" to="image_depth"/>
			<remap from="depth_registered/points" to="pointcloud2/points"/>
    </node>


	<node name="image_view" type="image_view" pkg="image_view" required="True" if="$(arg show_image)">
		<remap from="image" to="image_display"/>
	</node>

</launch>
